sqlmesh/utils/yaml.py

```diff
@@ -32,6 +34,7 @@ def load(
             source = file.read()
 
     if render_jinja:
```

Comment:
why are you mutating the constant?

Reply:
perhaps a new dict copy?

Reply:
Ah whoops, yeah I'll explode the constant within a new dict. Thanks for catching.